### ISSUE TITLE
Zone Select Screen

### ISSUE THREAD

**Issue by dgpt** [parkm/bpm-unity#7]
Area to navigate between various zones (quest hubs)

**Comment by dgpt** [comment]
It'd be cool to make this a map, with the option to go straight to the blacksmith or wizard's tower (I imagine these being on the outskirts of the town).

edit: I describe my envisioned layout of the game world in the quests_storyline file (see #1)

**Comment by parkm** [comment]
Yeah I really like this idea! You should make a small image draft of it. This can be focused on later though. I think the current drop down menu system on the 'quests' branch will be fine for now.
